src/Component/JsonSchemaCompiler/TypeGuesser/EnumGuesser.php

```diff
@@ -31,7 +37,7 @@ public function guessType(Registry $registry, JsonSchema $schema): ?Type
                     $valueType = Type::INTEGER;
                 }
                 if (\is_float($value)) {
-                    $valueType = Type::FLOAT;
+                    $valueType = Type::STRING;
```

Comment:
It's because BackedEnum can only return `string` or `int` ?

Reply:
yes 👍